jerry-core/CMakeLists.txt

```diff
@@ -143,13 +143,12 @@ else()
 endif()
 
 # ES5.1 profiles
-# Full profile
-if(FEATURE_PROFILE STREQUAL "full")
-  set(DEFINES_JERRY ${DEFINES_JERRY} CONFIG_ECMA_NUMBER_TYPE=CONFIG_ECMA_NUMBER_FLOAT64)
-
 # Compact profile
-elseif(FEATURE_PROFILE STREQUAL "compact")
-  set(DEFINES_JERRY ${DEFINES_JERRY} CONFIG_ECMA_COMPACT_PROFILE)
+if(FEATURE_PROFILE STREQUAL "compact")
+  set(DEFINES_JERRY
+  ${DEFINES_JERRY}
+  CONFIG_ECMA_COMPACT_PROFILE
+  CONFIG_ECMA_NUMBER_TYPE=CONFIG_ECMA_NUMBER_FLOAT32)
```

Comment:
Was this line added by mistake? (CONFIG_ECMA_NUMBER_TYPE=CONFIG_ECMA_NUMBER_FLOAT32)

This would give us:
   * Minimal profile: FLOAT64
   * Compact profile: FLOAT32
   * Full profile: FLOAT64

I think we want the default to be FLOAT64 in all three profiles :)